.gitignore

```diff
@@ -3,6 +3,7 @@
 .task/*
 sbom.*.json
 vulns.*.json
+seiso_goat_*.tar
```

Comment:
I wanted this to be a : between the image name and tag to align with docker convention but GitHub actions can't handle artifacts with : in them because it may use NTFS in the backend 😶